client/webserver/site/src/js/app.ts

```diff
@@ -849,6 +849,7 @@ export default class Application {
     if (order.matches) {
       for (let i = 0; i < order.matches.length; i++) {
         const match = order.matches[i]
+        console.log(`match.swap = ${JSON.stringify(match.swap)}`)
```

Comment:
Is the PR missing the key check on the revoked condition?

Reply:
I accidentally committed the wrong thing when I was busting the caches.